vcpkg.json

```diff
@@ -17,11 +17,11 @@
     "boost-asio",
     "boost-program-options",
     "boost-stacktrace",
-    "date",
```

Comment:
I think we can leave this in, it is tiny and really useful when working with dates.